tools/isobuild/compiler-plugin.js

```diff
@@ -776,8 +776,14 @@ export class PackageSourceBatch {
 
       map.forEach((info, name) => {
         if (! name) return;
+
+        let mainModule = _.find(info.files, file => file.mainModule);
+        mainModule = mainModule ?
+          `meteor/${name}/${mainModule.targetPath}` : false;
```

Comment:
This can just be `mainModule.installPath`, I think, since that includes `meteor/${name}/` prefix already.